### PR TITLE
include core version into js/css version tag

### LINKED ISSUE #13702: Start Update button does not work in Chrome running under Windows
### Steps to reproduce

1. Update Owncloud to 6 to 7 version
2. Update to 7.0.3 to 8.0.0 beta version in Windowsserver and with Chrome as browser
3. Click on Start Update button

### Expected behaviour
Update should start

### Actual behaviour

Start Update button does not work and there's also a small tiny text "owncloud" over the owncloud logo as it is shown in the screenshot attached.

### Server configuration
**Operating system**:
WindowsServer2008

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** (see ownCloud admin page)
{"installed":true,"maintenance":false,"version":"8.0.0.1","versionstring":"8.0 alpha 1","edition":""}

**Updated from an older ownCloud or fresh install:**
Updated from 7.0.3 to 8.0.0 beta

**List of activated apps:**
none

**The content of config/config.php:**

```
Insert your config.php content here
(Without the database password and passwordsalt)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** no

### Client configuration
**Browser:**
Chrome
captura de pantalla 2015-01-27 a las 9 29 32

### PR DESCRIPTION
**Pull request by DeepDiver1975**
fixes #13702

@PVince81 @MorrisJobke @nickvergessen  @SergioBertolinSG please review and test - THX

### CODE REVIEW

File: lib/private/templatelayout.php [hunk 1]
```diff
@@ -92,7 +92,9 @@ public function __construct( $renderAs, $appId = '' ) {
 
 
 		if(empty(self::$versionHash)) {
-			self::$versionHash = md5(implode(',', OC_App::getAppVersions()));
+			$v = OC_App::getAppVersions();
+			$v['core']= implode('.', \OC_Util::getVersion());
```
Comment (MorrisJobke) [inline lib/private/templatelayout.php]: missing ` ` before the `=`

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
Works :+1:

* checked out `v8.0.0beta1` -> c6... as hash
* checked out `master` -> same hash
* checked out this branch ->1b0b.. as hash -> success

File: lib/private/templatelayout.php [hunk 2]
```diff
@@ -92,7 +92,9 @@ public function __construct( $renderAs, $appId = '' ) {
 
 
 		if(empty(self::$versionHash)) {
-			self::$versionHash = md5(implode(',', OC_App::getAppVersions()));
+			$v = OC_App::getAppVersions();
+			$v['core'] = implode('.', \OC_Util::getVersion());
```
Comment (nickvergessen) [inline lib/private/templatelayout.php]: so core was missing he?

**Comment by nickvergessen** [comment]
:+1:

**Comment by SergioBertolinSG** [comment]
Tested #13702 on this branch, working fine.

**Pull request merged**